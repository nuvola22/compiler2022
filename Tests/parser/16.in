var a: Integer = 1;

begin 
writeln(a); 
end.