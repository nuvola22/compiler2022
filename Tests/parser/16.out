      program
         variable
            a
            Integer
            1
         begin
            call
               writeln
               a
